src/golang/lib/database/stmt_preparers/stmt_preparer.go

```diff
@@ -15,4 +21,18 @@ type StmtPreparer interface {
 	PrepareInsertWithReturnAllStmt(table string, columns []string, allColumns string) string
 	PrepareUpdateWhereStmt(table string, columns []string, predicateColumn string) string
 	PrepareUpdateWhereWithReturnAllStmt(table string, columns []string, predicateColumn string, allColumns string) string
+	// This helper method returns a query fragment that updates exec state blob
+	// with the given status and timestamp.
+	// This is useful to update the state without deserializing the content.
+	// Example: PrepareUpdateExecStateStmt('integration.execution_state', 'succeeded', time.Now())
+	// -> '`integration.execution_state = CAST(
+	//  json_set(json_set(integration.execution_state, '$.status', 'succeeded'),
+	//    '$.timestamps.finished_at', '2023-03-27 14:13PM'
+	//  ) AS BLOB)`
+	PrepareUpdateExecStateStmt(
```

Comment:
From a code organization standpoint, I would like to keep `lib/database` free of any Aqueduct application logic. If there is a query that is reused between different `sqlite repos`, lets create a shared helper underneath `repos/sqlite`.